src/core/filters/filterregistry.js

```diff
@@ -57,18 +56,65 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the filter string to send in a search query.
-   * TODO: move payload method logic into core.js, since it is only used there.
-   * @returns {string}
+   * Gets the static filters as a {@link CombinedFilter} to send to the answers-core
    */
   getStaticFilterPayload () {
-    return JSON.stringify(this._getStaticFilterPayload());
+    const filterNodes = this.getStaticFilterNodes();
+
+    const transformedFilterNodes = filterNodes.map(filterNode => {
+      return this._transformFilter(filterNode);
+    }).filter(filter => filter);
+
+    if (transformedFilterNodes.length > 1) {
+      return {
+        combinator: '$and',
+        filters: transformedFilterNodes
+      };
+    }
+    return transformedFilterNodes[0];
   }
 
-  _getStaticFilterPayload () {
-    const filterNodes = this.getStaticFilterNodes();
-    const totalNode = FilterNodeFactory.and(...filterNodes);
-    return totalNode.getFilter();
+  _transformFilter (filterNode) {
```

Comment:
Can we come up with a more descriptive name or add some JSDocs? It's not clear from the method signature what the result of the transform is. Potentially, we could name this `transformFilterNode` (as that's what is being transformed) and add some JSDoc about how it is parsed into either a `CombinedFilter` or `SimpleFilter`